alarms/Telegram/telegram_alarm.py

```diff
@@ -64,10 +70,28 @@ def set_alert(self, settings, default):
 		alert['title'] = settings.get('title', default['title'])
 		alert['body'] = settings.get('body', default['body'])
 		alert['location'] = parse_boolean(settings.get('location', default['location']))
+		alert['sticker_id'] = settings.get('send_stickers', default['sticker_id'])
 		return alert
  		
 	#Send Alert to Telegram
  	def send_alert(self, alert, info):
```

Comment:
add a parameter 'sticker=None' to the end of this. Allows us to specify a sticker_id